### PR TITLE
Add an example for calling the Pi-hole disable service

### PR DESCRIPTION
**Pull request by peca89**
## Proposed change

Add an example for calling the service to allow less experienced users to use it

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @peca89, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18945).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/pi_hole.markdown [hunk 1]
```diff
@@ -40,3 +40,25 @@ Disables configured Pi-hole(s) for the specified amount of time.
...
+
+You can add the following to your `configuration.yaml` file if want to create a template switch which will disable Pi-Hole temporarily as opposed to turning off the main `switch` component provided by the integration. You can turn the switch on before the timer expires for an immediate effect or just leave it to turn itself on after timer expires.
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: template
+    switches:
+      pihole_temp_disable:
+        friendly_name: "Disable Pi-Hole for 30 min"
```
Comment (frenck) [inline source/_integrations/pi_hole.markdown]: This feels weird to do, why not use an automation with a input_boolean for that?
Feel weird to have a template for this.

Additionally, please make sure the CI passes, thanks 👍

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/pi_hole.markdown (hunk 1 above)
Comment (peca89) [reply source/_integrations/pi_hole.markdown]: Frankly, I have no idea what input boolean is :) I just wanted to save beginners like me from spending 2 hours trying to call this service that has no example given. New users don't know how to format `timedelta` type and searching for "Home Assistant timedelta" gives mostly references to Python timedelta() function...Also, new users don't know how to format a service call. Forgive me, but I thought quick and dirty example is better than no example. And for me this achieves exactly what I wanted in terms of functionality. I'd be happy to see alternatives, but I don't know how to make them. 
Anyway, it seems that checks are now passing.

Comment (frenck) [reply source/_integrations/pi_hole.markdown]: > Frankly, I have no idea what input boolean is :) I just wanted to save beginners like me from spending 2 hours trying to call this service

Let make an example service call in that case, and not a template switch (which IMHO isn't a great example or a method I would use/recommend personally).

**Review by frenck — APPROVED** [review]
Thanks, @peca89 👍

**Pull request merged**